service/controller/clusterapi/v19/resources/chartconfig/desired.go

```diff
@@ -35,6 +35,12 @@ func (r *Resource) GetDesiredState(ctx context.Context, obj interface{}) (interf
 	var chartConfigs []*g8sv1alpha1.ChartConfig
 
 	for _, chartSpec := range r.newChartSpecs() {
+		// No ChartConfig CR is added because it has been migrated to use an
+		// App CR.
```

Comment:
We use the flag here so we don't create legacy chartconfig CRs for migrated apps.